scripts/release/uniprot_update/uniprot_xml2sql_isoform.pl

```diff
@@ -921,28 +919,33 @@ sub updateinstance {
             next;
         }
 
+        if (lc $attribute eq 'checksum' && is_sequence_changed_attribute_needs_updating($i, $new_values)) {
+            $changed = 1;
+            print $sequence_report_fh $i->displayName . ' (' . $i->db_id . ") has a new is_sequence_changed value\n";

```

Comment:
This is more of a question about things -- is 'is_sequence_changed' an attribute value?

Reply:
I see from the function below that it is -- resolving